src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,13 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args){
```

Comment:
There should not be a redundant empty space between the method signature and the opening brace. It should be `main(String[] args) {` instead of `main(String[] args){`.